### ISSUE TITLE
Edge opens with wrong address

### ISSUE THREAD

**Issue by nickserv** [MicrosoftEdge/MicrosoftEdgeLauncher#23]
When providing a URL and the `-k` flag, in some cases Edge will stay at example.com instead of switching the tab to the given URL. It seems like this could be a network related issue, since one of the users experiencing this was unable to access example.com, though I can access the website and I don't have this issue myself. Please see the comments of https://github.com/nickmccurdy/karma-edge-launcher/issues/27 for more information.

CC @gmisiolek @pchorus @leandro86

**Comment by rostacik** [comment]
hello gents. 

I can relate to this one. in my case , the solution from https://github.com/nickmccurdy/karma-edge-launcher/issues/27 doesn't even work, so I volunteer to repro and help you in any way.

my karma opens EDGE but I will not move further than this :

![image](https://cloud.githubusercontent.com/assets/429190/24252269/3fcd17b0-0fdd-11e7-8e0c-9011427ce324.png)

running something like this : MicrosoftEdgeLauncher.exe https://www.youtube.com/watch?v=fDrTbLXHKu8 -k

gives me this : Couldn't find Edge URL with URL: 01313080
Found

**Comment by mboughaba** [comment]
Hi everyone,
  
I started experiencing this issue myself. Edge launches `example.com` and hangs.

Cheers

**Comment by poman31** [comment]
Had this same issue today.  Edge would launch but since http://example.com does not resolve on our network, edge just sat there and edge-launcher reported it cannot start edge.   Had to change the hardcoded uri in the program and recompile.    Maybe we can add another parameter or change how this matches edge tabs.

**Comment by nickserv** [comment]
Does Edge have an equivalent to `about:blank` that would work offline as a workaround?

**Comment by poman31** [comment]
nickmccurdy:  I started up Edge without a url, and the application does not have an about:blank like IE.   The problem is that the launcher is using an exact match to a hardcode url to find the correct PID for the edge tab.   Maybe we can make the matching more robust to account for the DNS issue I was experiencing.  I need to check technet for various error states for edge.

**Comment by nickserv** [comment]
Good point. In that case, wouldn't it be easier to fix it so it starts with the correct URL initially (without example.com or an other placeholders)?

**Comment by jorgeibarra13** [comment]
Hi, I'm having this issue with my karma-edge-launcher, any tips on how to fix it?

**Comment by shreyavi** [comment]
I am facing the same issue, when I try to run karma start --browsers Edge. I am redirected to example.com which keeps on opening and closing and end with the following:

`05 08 2020 17:08:47.367:INFO [karma-server]: Karma v5.1.1 server started at http://localhost:9876/
05 08 2020 17:08:47.368:INFO [launcher]: Launching browsers Edge with concurrency unlimited
05 08 2020 17:08:47.375:INFO [launcher]: Starting browser Edge
05 08 2020 17:08:49.240:INFO [Edge 18.17763 (Windows 10)]: Connected on socket RAAAAEQWE89 with id 5345390
05 08 2020 17:08:49.688:ERROR [launcher]: Edge crashed.
        undefined
05 08 2020 17:08:49.688:ERROR [launcher]: Edge stdout:
Waiting for process 40420 to exit...
05 08 2020 17:08:49.690:ERROR [launcher]: Edge stderr:
05 08 2020 17:08:49.693:INFO [launcher]: Trying to start Edge again (1/2).
05 08 2020 17:08:51.651:WARN [Edge 18.17763 (Windows 10)]: Disconnected (0 times)reconnect failed before timeout of 2000ms (transport close)
Edge 18.17763 (Windows 10) ERROR
  Disconnectedreconnect failed before timeout of 2000ms (transport close)`

The point mentioned in [https://github.com/nickmccurdy/karma-edge-launcher/issues/27](url) regarding downloading edge-launcher 1.2.0.0, run MicrosoftEdgeLauncher.exe -k URL works fine with the format MicrosoftEdgeLauncher.exe URL -k. But yet can t run the unit tests.

It was working fine atleast locally till a month back but now the edge browser just keeps on crashing.

Please help on how to fix it.
